### PR TITLE
Add recipe for stashy

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
@conda-forge/staged-recipes @igortg @marcelotrevisani I think this is ready for review/merge. 👍

File: recipes/stashy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8137932f8fec2400e555fb230d51a47deb389955d2a2dbb37c1b6e59cf5f5983
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/stashy/meta.yaml]: Can this be noarch: python?

Comment (nicoddemus) [reply recipes/stashy/meta.yaml]: No, there's a preprocessor conditional for enum34 on the list of dependencies... :/

Comment (scopatz) [reply recipes/stashy/meta.yaml]: Uggh ok

**Pull request merged**